app/views/users/new.html.haml

```diff
@@ -1,15 +1,22 @@
 %section.register.container
   .row
-    .col-sm-10.col-sm-offset-1
-      = bootstrap_form_for @user,
-                           layout: :horizontal,
-                           label_col: 'col-sm-2',
-                           control_col: 'col-sm-10' do |f|
-        %header
-          %h1 Register
-        %fieldset
-          = f.email_field :email, label: "Email Address"
-          = f.password_field :password
-          = f.text_field :full_name,label: "Full Name"
-          = f.form_group do
-            = f.submit "Sign Up"
+    = form_for @user,layout: :horizontal do |f|
+      %header
+        %h1 Register
+      %fieldset
+        - if @token
+          = f.hidden_field :token, value: @token
+        .form-group.row
+          = f.label :email, "Email Address", class: 'col-form-label col-sm-2'
```

Comment:
Did this create any odd alignments? Usually a column contains elements, but the elements themselves are not columns. You can see this on your inputs such as lines 11-12.

It has been a while since I have worked with bootstrap, so this might be fine.